Data Protection Officer and minor formatting (#660)
chromaui/chromatic-docs: src/content/account/privacy-policy.md

We use the following types of cookies for the following purposes:

- **Strictly Necessary Cookies**: These cookies are essential for you to browse our Website and use its intended functionality, including accessing secure areas of our Website. These cookies cannot be opted-in or out of.

> - **Performance Cookies**: These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our Website. They help us to know which pages are the most and least popular and see how visitors move around our Website. All information these cookies collect is aggregated and therefore anonymous. If you do not allow these cookies, we will not know when you have visited our site.

Reviewer: Unless this is taken from a legal document and not possible to change, we could probably go with:
```suggestion
- **Performance Cookies**: These cookies allow us to count visits and traffic sources to measure and improve our Website's performance. They help us know which pages are the most and least popular and see how visitors move around our Website. All information these cookies collect is aggregated and, therefore, anonymous. If you do not allow these cookies, we will not know when you have visited our site.
```

**Controller.**  We are the controller of your personal information covered by this Privacy Policy for the purposes of European data protection legislation, except to the extent that we process your personal information on behalf of our customer, including personal information that we process on behalf of our Clients, in which case our customer is the controller of your personal information, and we are the processor.

We have appointed a Data Protection Officer, whose contact information is:
```
DP Dock DPO Services GmbH
Grüffkamp 10
24159 Kiel
Germany
chromatic@dp-officer.com 
```

> 

Reviewer: When you can, can you check the table below? It doesn't seem right to me. Specifically the 
**Processing Purpose**'s 2nd and 3rd rows

Author: Thanks, this is the correct format according to the internet
